resolver/serviceResolver.go

```diff
@@ -571,7 +571,7 @@ func (resolver *serviceResolver) checkAllowanceAndVerifyCode(
 
 	return &requests.OTPCodeVerifyData{
 		RemainingTrials:             int(resolver.secureOtpHandler.FreezeMaxFailures()),
-		ResetAfter:                  0,
+		ResetAfter:                  -1,
```

Comment:
i think we can still keep ResetAfter as 0, it is not certain at this point that the key does not have a ttl